router/pkg/authentication/jwks_token_decoder.go

```diff
@@ -46,16 +47,26 @@ type JWKSConfig struct {
 	Secret    string
 	Algorithm string
 	KeyId     string
-}
 
-func NewJwksTokenDecoder(ctx context.Context, logger *zap.Logger, configs []JWKSConfig) (TokenDecoder, error) {
+	Audiences []string
+}
 
-	remoteJWKSets := make(map[string]jwkset.Storage)
+type AudKey struct {
```

Comment:
I think this type doesn't have to be exported as it isn't used anywhere else

Reply:
Done